ingestion_tools/scripts/data_validation/standardized/tests/test_tiltseries.py

```diff
@@ -22,12 +26,20 @@ def set_helper_test_mrc_zarr_header_class_variables(
         tiltseries_zarr_metadata: Dict[str, Dict[str, Dict]],
         tiltseries_metadata: Dict,
     ):
-        self.spacegroup = 0  # 2D image
         self.mrc_headers = tiltseries_mrc_header
         self.zarr_headers = tiltseries_zarr_metadata
         self.spacing = tiltseries_metadata["pixel_spacing"]
         self.skip_z_axis_checks = True
 
+    @pytest.fixture
+    def tiltseries_metadata_range(self, tiltseries_metadata: dict) -> list[float]:
+        # add tilt_step to max because arange is end value exclusive
+        return np.arange(
+            tiltseries_metadata["tilt_range"]["min"],
+            tiltseries_metadata["tilt_range"]["max"] + tiltseries_metadata["tilt_step"],
+            tiltseries_metadata["tilt_step"],
+        ).tolist()
```

Comment:
Seems this could go in the tiltseries helper?

Reply:
Could you elaborate on this?

Reply:
oh, this method just looks like a helper method that could go into TiltSeriesHelper. not a big deal